exec/java-exec/src/main/java/org/apache/drill/exec/record/metadata/AbstractColumnMetadata.java

```diff
@@ -55,6 +72,21 @@
    */
 
   protected int expectedElementCount = 1;
+  protected Map<String, String> properties = new HashMap<>();
```

Comment:
This field can be made final. Also, In some places values from `LinkedHashMap` are put into this map. Should it be also `LinkedHashMap`?